tests/E2E/OrganizationTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * This file is part of the contentful-management.php package.
+ *
+ * @copyright 2015-2017 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\Tests\E2E;
+
+use Contentful\Management\Query;
+use Contentful\Tests\End2EndTestCase;
+
+class OrganizationTest extends End2EndTestCase
+{
+    /**
+     * @vcr e2e_organization_get_collection.json
+     */
+    public function testGetOrganizations()
+    {
+        $organizations = $this->client->getOrganizations();
+
+        $this->assertCount(3, $organizations);
+
+        $organization = $organizations[2];
```

Comment:
Unless someone adds that user to another org it shouldn't break 😄

We could see if orderBy works on that endpoint to make sure the order doesn't change, I'll check that later.